### PR TITLE
Configure automated EAP releases with github actions

### PR DESCRIPTION
**Pull request by timaliberdov**

### CODE REVIEW

File: .github/workflows/publish-eap.yml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+      - name: Package plugin
+        # The first packageArtifact creates the files (including plugin.xml).
+        # The second packageArtifactZip allows the patchPluginXML task to run
+        #   on those files and patch the plugin.xml.
+        # Then it packages files into a single zip archive.
+        run: sbt clean && sbt packageArtifact && sbt packageArtifactZip
+
+      - name: Publish plugin
+        run: sbt publishPlugin eap
```
Comment (hmemcpy) [inline .github/workflows/publish-eap.yml:71]: Slight nitpick, but I think it uses the name as the display name of the channel. Maybe make it `EAP` in ALL CAPS?

Comment (timaliberdov) [reply .github/workflows/publish-eap.yml:71]: I guess, this channel name is up to us: https://plugins.jetbrains.com/docs/marketplace/custom-release-channels.html#configuring-a-custom-channel-in-intellij-platform-based-ides

> For example, if you specify 'eap' as a release channel, then users will need to add the https://plugins.jetbrains.com/plugins/eap/list repository to install the plugin and receive updates. These channels are treated as separate repositories for all intents and purposes.

But if you think that EAP is better than eap, I'll fix it :)

Comment (hmemcpy) [reply .github/workflows/publish-eap.yml:71]: nah, I just read the doc. I think `eap` is fine :)

**Comment by hmemcpy** [comment]
BOOM!
This... this is amazing :)

Can we manually trigger the EAP publish?

**Comment by timaliberdov** [comment]
Sure, this `workflow_dispatch:` is just for that!

**Comment by timaliberdov** [comment]
Just smash that 'Run workflow' button in 'Actions' tab! :D 
image

**Pull request merged**